src/framed/std/core.cljc

```diff
@@ -121,10 +122,11 @@
   ([f y x]
    (f x y)))
 
-(defmacro future-loop
-  "Execute body repeatedly within a future, returning the future"
-  [& body]
-  `(future
-     (loop []
-       ~@body
-       (recur))))
+#?(:clj
+  (defmacro future-loop
```

Comment:
This would have been a good thing to test before OK the code review and committing...